README.md

```diff
@@ -46,10 +46,18 @@ $ ./hubble-cli "/Users/hubble/Desktop/MyDesign.sketch"
 
 # You can optionally specify a config output dir. Otherwise current working directory will be used:
 $ ./hubble-cli "/Users/hubble/Desktop/MyDesign.sketch" --outputDir="/var/hubble/"
+
+# You can also export to a Style Dictionary compatible format.
+$ ./hubble-cli "/Users/hubble/Desktop/MyDesign.sketch" --useStyleDictionaryOutput
 ```
 
 If you need more fine-tuning over this export flow, Hubble provides even more options to [customize your design token output][wiki].
 
+### Style Dictionary
+
+Hubble Scripts supports a variant on our standard ITP Tokens, which is compatible with the [style dictionary][style dictionary] build system.
```

Comment:
Please replace:

```suggestion
hubble-scripts supports a variant on our default generic design tokens, which is compatible with the [Style Dictionary][style dictionary] build system.
```

Reply:
+ link to https://github.com/inthepocket/hubble-scripts/wiki/Exporting-design-tokens#style-dictionary